Tasks/KubernetesManifestV0/src/utils/KubernetesObjectUtility.ts

```diff
@@ -239,7 +231,11 @@ function getImagePullSecrets(inputObject: any) {
         return null;
     }
 
-    if (isEqual(inputObject.kind, KubernetesWorkload.Pod, StringComparer.OrdinalIgnoreCase)) {
+    if (isEqual(inputObject.kind, KubernetesWorkload.cronjob, StringComparer.OrdinalIgnoreCase)) {
+        return inputObject.spec.jobTemplate.spec.template.spec.imagePullSecrets;
```

Comment:
null check for jobTemplate required? Also where is the handling for Workload type - job ?

Reply:
Job has a default spec template. It gets handled in the default workload `if..else` case.

Reply:
Surrounded the fetching and overriding logic with a `try..catch` block